style.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap');
```

Comment:
Your css looks great! Amazing job on watching out for redundancies and reusing your rules 👌✔😵